### PR TITLE
Backend yue

### PR DESCRIPTION
**Pull request by Yue0218**
calculation page

### CODE REVIEW

File: LawnCare/android/app/src/main/java/com/redflag/lawncare/MainActivity.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ void setTabsAndViewPager() {
             tabLayout.addTab(tabLayout.newTab().setText("Book"));
             tabLayout.addTab(tabLayout.newTab().setText("FAQ"));
             tabLayout.addTab(tabLayout.newTab().setText("Contact"));
+            tabLayout.addTab(tabLayout.newTab().setText("Quote"));
```
Comment (wrm192) [inline LawnCare/android/app/src/main/java/com/redflag/lawncare/MainActivity.java:45]: These labels should be translated. Something we missed in the last version. I will add a ticket in the future to fix these

File: LawnCare/android/app/src/main/java/com/redflag/lawncare/calculation/CalculationFragment.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_calculation,
+                container, false);
+
+
+        Button submit = view.findViewById(R.id.SubmitButton);
+        submit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                TextInputLayout input1_TIL = view.findViewById(R.id.input_1);
```
Comment (wrm192) [inline LawnCare/android/app/src/main/java/com/redflag/lawncare/calculation/CalculationFragment.java:28]: Might be nit picking here, but i do think we should add more descriptive names to R.id.input_1. Like r.id.area or obstruction for example, it'll save us a potential headache in the future I think.

File: LawnCare/android/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -33,5 +33,11 @@
     <string name="perf_contact_choice">Preferred method of contact:</string>
     <string name="perf_contact_choice_email">Email</string>
     <string name="perf_contact_choice_phone">Phone</string>
+    <string name="calculation_title">Estimated Price</string>
+    <string name="input_1">Area</string>
```
Comment (wrm192) [inline LawnCare/android/app/src/main/res/values/strings.xml:37]: same with here, Area  instead of Input_1 please.

**Comment by wrm192** [comment]
This is good Yue, thank you

File: LawnCare/android/app/src/main/java/com/redflag/lawncare/calculation/CalculationFragment.java (hunk 2 above)
Comment (Yue0218) [reply LawnCare/android/app/src/main/java/com/redflag/lawncare/calculation/CalculationFragment.java]: fixed!

File: LawnCare/android/app/src/main/java/com/redflag/lawncare/MainActivity.java (hunk 1 above)
Comment (Yue0218) [reply LawnCare/android/app/src/main/java/com/redflag/lawncare/MainActivity.java:45]: I couldnt find where the tabs id are, will add them later when I find it! Let me know if you know~

**Pull request merged**